crates/forge/tests/fixtures/replay_last_run_failures.stdout

```diff
@@ -0,0 +1,15 @@
+No files changed, compilation skipped
+
+Ran 2 tests for test/ReplayFailures.t.sol:ReplayFailuresTest
+[FAIL. Reason: revert: testB failed] testB() (gas: 303)
+[FAIL. Reason: revert: testD failed] testD() (gas: 314)
+Suite result: FAILED. 0 passed; 2 failed; 0 skipped; finished in 555.95µs (250.31µs CPU time)
+
+Ran 1 test suite in 3.24ms (555.95µs CPU time): 0 tests passed, 2 failed, 0 skipped (2 total tests)
```

Comment:
Should this be "2 tests passed, 2 failed, 0 skipped (4 total tests)"?

Reply:
ah, that's not the initial run but only the 2nd run (failures only) output asserted in test, for initial run test just checks there are failures

Reply:
Ah I see :+1: